integration_tests/test_emigration.py

```diff
@@ -0,0 +1,61 @@
+import pandas as pd
+
+
+def test_individuals_emigrate(simulants_on_adjacent_timesteps):
+    # This is a very rare event, so we can't assert that it happens
+    # on every timestep; instead, we aggregate across all timesteps.
+    all_individual_emigration_status = []
+
+    for before, after in simulants_on_adjacent_timesteps:
+        individual_emigrants = (
+            before["tracked"]
+            & ~after["tracked"]
+            & after["household_id"].isin(after[after["tracked"]]["household_id"])
+        )
+
+        # Only occurs to living, tracked people
+        living_tracked = before["tracked"] & (before["alive"] == "alive")
+        assert living_tracked[individual_emigrants].all()
+
+        assert after["time"].nunique() == 1
+        current_time = after["time"].iloc[0]
+        assert (after[individual_emigrants]["exit_time"] == current_time).all()
+
+        all_individual_emigration_status.append(individual_emigrants[living_tracked])
+
+    all_individual_emigration_status = pd.concat(
+        all_individual_emigration_status, ignore_index=True
+    )
+    # VERY conservative upper bound on how often this should be occurring, as a proportion
+    # of living, tracked simulant-steps
+    assert 0 < all_individual_emigration_status.mean() < 0.1
```

Comment:
The input data to the simulation is stratified by demographics and location (US state). The "ideal" population-level value will depend on the population structure with respect to those variables.

I noted that this is `VERY conservative` above: in no case is the emigration rate above 0.05 *per year*. While I'd love to test the rate more directly, it is not trivial to do so in a way that is robust to stochastic variation and population size.

If you want me to make this less conservative without fundamentally changing the approach, i.e. there is another "magic number" you'd prefer to use here instead of 0.1, I am happy to do that.